runtime/src/main_launcher.c

```diff
@@ -186,6 +186,40 @@ chpl_run_utility1K(const char *command, char *const argv[], char *outbuf, int ou
 }
 
 //
+// This is an even simpler run-a-command utility.  The command is just
+// a string to be run, by something like `/bin/sh -c "commandStr"`.  It
+// must not expect any input.  Its output to stdout is placed in outbuf,
+// truncated to fit if necessary.  Its output to stderr is discarded.
+// On success, the return value is the number of bytes in outbuf.  On
+// failure, it is -1; output may have been placed in outbuf in this
+// case, but if so it should be ignored.
+//
+int
+chpl_run_cmdstr(const char *commandStr, char *outbuf, int outbuflen) {
```

Comment:
Why do you need to add this instead of just using `chpl_run_utility1K()`?

Reply:
The interface to chpl_run_utility1K() seemed unnecessarily complicated because of the argv[] vector.  In many cases all that's needed is just to give a command string to a shell, which is what this does.

Reply:
I have a mild preference for using the other function since it's more tested. That's not a great reason though, so I'm fine with keeping this